pages/account/index.md

```diff
@@ -1,11 +0,0 @@
----
-title: Your VA.gov Account
-layout: page-react.html
-entryname: account
----
-<nav aria-label="Breadcrumb" aria-live="polite" class="va-nav-breadcrumbs" id="va-breadcrumbs">
-  <ul class="row va-nav-breadcrumbs-list columns" id="va-breadcrumbs-list">
-    <li><a href="/" onClick="recordEvent({ event: 'nav-breadcrumb', 'nav-breadcrumb-section': 'home' });">Home</a></li>
-    <li><a aria-current="page" href="/account/">Your account settings</a></li>
-  </ul>
-</nav>
```

Comment:
Migrated here, https://github.com/department-of-veterans-affairs/vets-website/pull/10660/files#diff-a63590cf49f70f7842572210d5da4934R7.

This app does _not_ have an E2E test.